x-pack/plugins/observability/common/index.ts

```diff
@@ -12,6 +12,9 @@ export {
   asPercent,
   getDurationFormatter,
   asDuration,
+  asDynamicBytes,
```

Comment:
Since these functions are now being used by another application other than Observability, it makes sense to start moving them to Observability Shared.

Reply:
thanks @CoenWarmer